lib/qbwc/request/products.rb

```diff
@@ -143,9 +143,13 @@ def polling_current_items_xml(params, config)
             return build_polling_from_config_param(params, session_id, time)
           end
 
+          if params['quickbooks_max_returned'] && params['quickbooks_max_returned'] != ""
+            inventory_max_returned = params['quickbooks_max_returned']
```

Comment:
.to_i? I don't know if it's needed here or not?

Reply:
It's all a string I guess right?

Reply:
We don't need to_i but this block would probably break now that I think of it due to undefined local variable. I'll push an update